### DISCUSSION TITLE
Auto notifications for changes to specific files

### DISCUSSION THREAD

**Discussion by jsalant22** [ansys/pyaedt#2725] [Q&A] [answered]
Is there a way to get automatic notifications if specific files are changed? Even better would be an option to automatically have someone from our team as an (optional) reviewer for any PRs that modify certain files.

**Comment by MaxJPRey** [comment, marked as answer]
@jsalant22 Yes, it is possible and often recommended, especially for specific area of the code requiring physics knowledge.
Please send me the list of the files and associated reviewers and I will set it up.

**Reply by jsalant22** [reply]
@MaxJPRey For now, add myself and @myoung301 as reviewers for these files:

test_26_emit.py
pyaedt/emit.py
pyaedt/emit_core (all files in here)
pyaedt/modeler/circuits/PrimitivesEmit.py
examples/07-EMIT (all files in here)

Thanks.

**Reply by MaxJPRey** [reply]
@jsalant22 I will notify you as soon as it is done.

**Reply by MaxJPRey** [reply]
@jsalant22 It is done. You and @myoung301 should be added as reviewers automatically when those files are modified.

**Reply by jsalant22** [reply]
@MaxJPRey I'm not sure what happened, but we are no longer getting the automatic notifications for these files. We did just need to "reauthorize" our access to PyAnsys so maybe the permissions got reset somehow?

cc: @myoung301

**Reply by MaxJPRey** [reply]
@jsalant22 I can see that @myoung301 is not part of the PyAEDT collaborators anymore.

This could be due to the GitHub organization renaming (PyAnsys became Ansys).

Could the 2 of you join the ansys organization, just to be sure: https://github.com/ansys
Then, click on [Join Ansys GitHub Organization](https://github.com/orgs/ansys/sso)
![image](https://github.com/ansys/pyaedt/assets/87315832/1fd7b219-dcbf-44ed-adcf-adaf635992ca)

**Reply by myoung301** [reply]
@MaxJPRey My account page shows I'm part of the organization again. Hopefully you're able to configure things now, but let me know if things still don't seem right with my account.

**Reply by MaxJPRey** [reply]
It should be fine now.
I just created #3027 for testing purpose.
you should be notified by now of its creation.
